build.gradle

```diff
@@ -1,9 +1,11 @@
 buildscript {
 	repositories {
 		maven { url "https://repo.spring.io/plugins-release" }
+		maven { url "https://repo.spring.io/plugins-snapshot" }
 	}
 	dependencies {
 		classpath("io.spring.gradle:propdeps-plugin:0.0.9.RELEASE")
+		classpath("io.spring.nohttp:nohttp-gradle:0.0.1.BUILD-SNAPSHOT")
```

Comment:
Is there a timeline for a release?

Reply:
I'm a bit concerned about baking such a very early-state plugin into existing builds, so I'd rather only have it in master for the 5.2 generation. As for the PR overall, the missing Apache license headers all affect new files in 5.2, so quite a bit seems to be master-specific to begin with. The only part I see as a 5.1 backport candidate are the URL examples in the reference docs.

Reply:
I expect to release it in the next week, but am wanting to get feedback from projects before that release is done.